projects/uitdatabank/models/common-title.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "copyrightHolder",
```

Comment:
This file is called `common-title` but here it is called `copyrightHolder`. This is a new file but nowhere used?

Reply:
You are right, it does not do anything. it was a mistake to commit this file.
I have removed it.